.gitignore

```diff
@@ -31,3 +31,4 @@ BenchmarkDotNet.Artifacts/
 korebuild-lock.txt
 .gradle/
 src/SignalR/clients/**/dist/
+modules/
```

Comment:
/cc @sebastienros

Reply:
I kept accidentally committing submodules when switching between release/2.x and master branches.